src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKPainter.java

```diff
@@ -213,17 +213,6 @@ public void paintRadioButtonMenuItemBackground(SynthContext context,
         if (gtkState == SynthConstants.MOUSE_OVER) {
             if (GTKLookAndFeel.is3()) {
                 paintComponentBackground(context, g, x, y, w, h);
```

Comment:
So do we really need the is3() check ?? Seems like not to me. Same for the case below at line 573.

Reply:
It is not the only place, I doubted it, but now updated to remove all usages.
However, prefer to keep the `is3` method itself.